### PR TITLE
A few nits

### LINKED ISSUE #87: Add links to HTML table views to GitHub pages
Arises from #75

### PR DESCRIPTION
**Pull request by znichollscr**
Fixes based on comment here https://github.com/PCMDI/input4MIPs_CVs/issues/90#issuecomment-2293474152

Closes #87 

## Checklist

Please confirm that this pull request has done the following:

- [x] Documentation added (where applicable)
- [x] Changelog item added to `changelog/`

### CODE REVIEW

**Comment by znichollscr** [comment]
Alrighty @durack1 all the background is in the description. This is ready for you to take a look and merge if you're happy.

File: python-packages/input4MIPs-CVs/src/input4MIPs_CVs/html_generation.py [hunk 1]
```diff
@@ -52,6 +52,13 @@ def get_url_esgf_for_html_table(row: pd.Series, search_facets: Iterable[str]) ->
     if publication_status == "abandoned":
         return "Abandoned: registered but never produced"
 
+    if publication_status == "never_published":
+        return (
+            "Never published: registered but never published on ESGF. "
+            "Note: this does not mean "
+            "that the dataset wasn't published on a platform other than  the ESGF."
```
Comment (durack1) [inline python-packages/input4MIPs-CVs/src/input4MIPs_CVs/html_generation.py:59]: extra space `than  the ESGF`

File: docs/input4MIPs_source-id_CMIP6Plus.html [hunk 2]
```diff
@@ -201,28 +218,15 @@
...
-      <td>CMIP6Plus</td>
-      <td>SOLARIS-HEPPA-CMIP-4-1</td>
-      <td>None</td>
-      <td>bernd@iaa.es</td>
-      <td>None</td>
-      <td>http://solarisheppa.geomar.de/cmip7</td>
-      <td>SOLARIS-HEPPA</td>
-      <td>CC BY 4.0</td>
-      <td>abandoned</td>
-      <td>4.1</td>
```
Comment (durack1) [inline docs/input4MIPs_source-id_CMIP6Plus.html:224]: Is there a reason why we lost this? 4.1 is a dataset that NCAR has been using in prototype simulations - it'd be better to keep it

File: docs/input4MIPs_source-id_CMIP6Plus.html [hunk 3]
```diff
@@ -22,20 +22,26 @@
 </head>
 <body>
 <p><h1>Input4MIPs CMIP6Plus source IDs: v6.5.20a1</h1><p>
```
Comment (durack1) [inline docs/input4MIPs_source-id_CMIP6Plus.html:24]: as a nit, I always like to use "input" rather than "Input" but now I am devolving into pedantry

**Review by durack1 — APPROVED** [review]
@znichollscr looks good, want me to layer over the fixes in `[input4MIPs_CVs/python-packages/input4MIPs-CVs/src/input4MIPs_CVs/cli/update-database.py](https://github.com/PCMDI/input4MIPs_CVs/blob/1c9c0b34e25ebe199a7dfbb78bf0e9a2237d6108/python-packages/input4MIPs-CVs/src/input4MIPs_CVs/cli/update-database.py#L206) as per #94 and then we merge and run "Bump version"?

File: python-packages/input4MIPs-CVs/src/input4MIPs_CVs/html_generation.py (hunk 1 above)
Comment (znichollscr) [reply python-packages/input4MIPs-CVs/src/input4MIPs_CVs/html_generation.py:59]:
```suggestion
            "that the dataset wasn't published on a platform other than the ESGF."
```

File: docs/input4MIPs_source-id_CMIP6Plus.html (hunk 3 above)
Comment (znichollscr) [reply docs/input4MIPs_source-id_CMIP6Plus.html:24]: Easy fix, will make it here too

File: docs/input4MIPs_source-id_CMIP6Plus.html (hunk 2 above)
Comment (znichollscr) [reply docs/input4MIPs_source-id_CMIP6Plus.html:224]: Hmm that shouldn't have happened, good pick up

Comment (znichollscr) [reply docs/input4MIPs_source-id_CMIP6Plus.html:224]: Fixed now I believe.

**Comment by znichollscr** [comment]
> want me to layer over the fixes in [`input4MIPs_CVs/python-packages/input4MIPs-CVs/src/input4MIPs_CVs/cli/update-database.py`](https://github.com/PCMDI/input4MIPs_CVs/blob/1c9c0b34e25ebe199a7dfbb78bf0e9a2237d6108/python-packages/input4MIPs-CVs/src/input4MIPs_CVs/cli/update-database.py#L206) as per #94 

Sure. Either here in a separate MR is fine.

> and then we merge and run "Bump version"?

Sounds good to me

**Comment by durack1** [comment]
Ok @znichollscr it seems this is good to go, no? and I have #99 is your review queue (there are CI issues raised). Once these are both green and merged then "Bump version" and we're onward and upward.

Let me know if this is good to go...

**Comment by znichollscr** [comment]
Yep good to go

**Pull request merged**